### DISCUSSION TITLE
PG CVs higher than DIANN

### DISCUSSION THREAD

**Discussion by ZCohenBPM** [MannLabs/alphadia#308] [General]
Analyzing the same diaPASEF data with the same spectral library, median CVs for DIANN pg quantification are 6.5, while from alphaDIA are 15.5. Both sets of data are from the pg_matrix files. I am using normalize LFQ, and there is only 3% CV on total protein amount for alphaDIA output compared to 1.5% for DIANN, so that doesn't seem to be the root cause. Is there a known reason for this discrepancy? CV histograms attached below.

![DIANN_CV_hist](https://github.com/user-attachments/assets/4b8c828e-9d78-4a42-a158-6e4775bdbac0)
DIANN 1.8

![alphadia_CV_hist](https://github.com/user-attachments/assets/7ba1389d-9b1d-47ed-97ee-d3a96995a0b1)
alphaDIA 1.7.2

**Comment by GeorgWa** [comment]
Hi, thanks for the report!

Can. you describe the data in terms of instrument, gradient length, MS2 method?
Did you use the same processing settings like MS1/MS2 tolerance?

**Comment by ZCohenBPM** [comment]
Data is from a TimsTof HT, 30 minute gradient, CID. MS1/MS2 Tolerance were both set to 10 ppm in both searches.

**Comment by GeorgWa** [comment]
This sounds reasonable. 

You can try increasing the target mobility tolerance and target rt tolerance, although I would be surprised to see such a large effect. We will introduce automatic optimization with the next version, which should solve this.

We are currently focussing on the Astral plattform but we have usually seen better CVs than this also on the timsTOF platforms.
